src/widgetastic/widget.py

```diff
@@ -1038,8 +1038,9 @@ class Table(Widget):
     Args:
         locator: A locator to the table ``<table>`` tag.
     """
-    HEADERS = './thead/tr/th|./tr/th'
     ROWS = './tbody/tr[./td]|./tr[not(./th) and ./td]'
+    HEADER_IN_ROWS = './tbody/tr/th'
```

Comment:
Hmm, I would even do so it would pick only the first tr in this case, so `./tbody/tr[1]/th`, what would you think? I don't think that we should support all kinds of bad design, developers should fix it :)